tests/test_unit_testing_structure.py

```diff
@@ -0,0 +1,14 @@
+from aioca import caget
+from fixtures.mocked_panda import TEST_PREFIX
+
+
+def test_conftest_loads_fixtures_from_other_files(table_fields):
+    ...
+
+
+async def test_fake_panda_and_ioc(mocked_panda_standard_responses):
```

Comment:
I like this. Can you just add a note/description to this test (or the whole file) to explain that it is just testing that the test framework itself works?